### ISSUE TITLE
needs 

### ISSUE THREAD

**Issue by revolunet** [SocialGouv/kontinuous#391]
Looks like in some case the needs are not honored

values.yaml
```yaml
app:
  ~chart: app
  ~needs: [build-app]
```

dev/values.yaml
```yaml
pgweb-hasura:
  ~chart: pgweb
  ~needs: [app]
```

In this example, pgweb will be deployed without waiting for app to be ready

Example in this branch : https://github.com/SocialGouv/template/tree/empty

<img width="962" alt="Capture d’écran 2023-08-03 à 12 58 30" src="https://github.com/SocialGouv/kontinuous/assets/124937/86237c81-e513-4405-b2f2-7083728f1eab">

**Comment by devthejo** [comment]
will be fixed by #392

**Comment by revolunet** [comment]
![](https://media.giphy.com/media/26gsjCZpPolPr3sBy/giphy.gif)
